node/peer.js

```diff
@@ -53,6 +53,7 @@ function TChannelPeer(channel, hostPort, options) {
         random: self.channel.random,
         period: self.options.period,
         maxErrorRate: self.options.maxErrorRate,
+        minimumRequests: self.options.minimumRequests,
```

Comment:
Naming nit: max is to min as maximum is to minimum. Consistency is my hobgoblin.